apps/trading/pages/_app.page.tsx

```diff
@@ -25,47 +25,49 @@ function AppBody({ Component, pageProps }: AppProps) {
   const store = useGlobalStore();
   const { VEGA_NETWORKS } = useEnvironment();
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const [_, toggleTheme] = useThemeSwitcher();
+  const [theme, toggleTheme] = useThemeSwitcher();
```

Comment:
praise (non-blocking): good catch! amazing. can't wait to see this merged.

Reply:
Thanks! :)